src/org/jitsi/impl/neomedia/rtp/FrameDesc.java

```diff
@@ -93,6 +103,18 @@
     }
 
     /**
+     * Gets a boolean that indicates whether or not we can read the frame
+     * boundaries of this frame.
+     *
+     * @return true if we're able to read the frame boundaries of this frame,
+     * false otherwise.
+     */
+    boolean supportsFrameBoundaries()
```

Comment:
Yeah, is that gonna throw an NPE if called before #supportsFrameBounderies is re-initialized with a non-null?